sdks/java/io/cdap/build.gradle

```diff
@@ -38,26 +38,35 @@ interface for integration with CDAP plugins."""
  */
 
 dependencies {
-    implementation library.java.avro
     implementation library.java.cdap_api
     implementation library.java.cdap_api_commons
     implementation (library.java.cdap_common) {
-        exclude module: "log4j-over-slf4j"
+        exclude module: 'log4j-over-slf4j'
     }
     implementation library.java.cdap_etl_api
     implementation library.java.cdap_etl_api_spark
+    implementation library.java.cdap_hydrator_common
+    implementation library.java.guava
     implementation library.java.hadoop_common
-    implementation library.java.hadoop_mapreduce_client_core
+    implementation library.java.hadoop_mapred
     implementation library.java.jackson_core
     implementation library.java.jackson_databind
-    implementation library.java.guava
     implementation library.java.slf4j_api
     implementation library.java.tephra
+    implementation library.java.vendored_guava_26_0_jre
     implementation project(path: ":sdks:java:core", configuration: "shadow")
+    implementation project(":sdks:java:io:hadoop-format")
+    implementation "io.cdap.plugin:salesforce-plugins:1.4.0"
```

Comment:
should these be moved up to the beam module plugin?

Reply:
Done